### DISCUSSION TITLE
subway access

### DISCUSSION THREAD

**Discussion by uegert** [streetcomplete/StreetComplete#4156] [Ideas]
Hi, I'm currently adding some info in Stockholm and find the subway access buildings difficult to address in StreetComplete. In StreetComplete, they are sometimes associated with incomplete building descriptions (for example [this one](https://www.openstreetmap.org/way/163123430)) but that category doesn't quite fit since in many cases they are essentially just covering the stairs, like the one linked. It is not clear to me if the question refers to just the roof or the whole entity subway_entrance. In particular if the subway_entrance is otherwise "complete", it can be difficult to know what is being asked for. 
They should have their own category in the buildings drop down list with several specific properties that would be useful (in my opinion):
subway station they belong to
escalators up/down/ on demand direction/none
elevator access
covered/closed/open
lighting
marking for blind people
handrails
wheelchair ramp 
wheelchair access
name, such as one referring to a specific road or compass direction
closing times (if any)
sometimes the elevator is a separate building next to the one with stairs, sometimes included in the entrance. 

Only some of these are addressed in the OSM-wiki for the Tag:railway=subway_entrance description. I'm not familiar enough with OSM to tell if all the listed properties can even be specified with this tag.

**Comment by matkoniecz** [comment]
https://www.openstreetmap.org/way/163123430 - for this things I would use answers "roof" or "public transportation building" or leave note
